### PR TITLE
feat: SEO-16 Canonical tag optimization docs

### PR DESCRIPTION
**Pull request by Gustavo341**

### CODE REVIEW

File: theme.config.tsx [hunk 1]
```diff
@@ -13,8 +13,11 @@ const config: DocsThemeConfig = {
     const { asPath } = useRouter()
     if (asPath !== '/') {
       return {
-        titleTemplate: '%s – Ankr'
+        titleTemplate: '%s – Ankr',
+        canonical: `https://www.ankr.com/docs${asPath}`
       }
+    } return {
+      canonical: 'https://www.ankr.com/docs/'
```
Comment (BigBadAlien) [inline theme.config.tsx:20]: @Gustavo341 Please leave a comment with samples regarding how does it work

File: theme.config.tsx [hunk 2]
```diff
@@ -13,8 +13,11 @@ const config: DocsThemeConfig = {
     const { asPath } = useRouter()
     if (asPath !== '/') {
       return {
-        titleTemplate: '%s – Ankr'
```
Comment (BigBadAlien) [inline theme.config.tsx:16]: // Description why do we need it
// Samples

Comment (BigBadAlien) [reply theme.config.tsx:16]: What we had and what are we having in the result

Comment (BigBadAlien) [reply theme.config.tsx:16]: Could you also add link to the docs

**Review by raproid — APPROVED** [review]

**Pull request merged**